### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by HelmiRifai** [metabase/metabase#2608]
Hey,

I'm facing an issue with (saved) questions occasionally with the below happening:

![image](https://cloud.githubusercontent.com/assets/4684370/15290837/a5b2d64a-1b83-11e6-8919-4dfd6b730460.png)

If this is a bug report, please fill in the blanks:
-  I am using the Chrome browser.
-  My computer's OS is Mac OSX.
-  I'm running Metabase AWS Docker Image
-  My database is Redshift.
-  My Metabase version is 0.17.0

**Comment by agilliland** [comment]
Hmmm, can you check your browser console and see if there is any error reported there when you see this happen?  You definitely should never end up with a blank Table when loading an existing question.

**Comment by HelmiRifai** [comment]
It's quite reproducible with me :D 

There you go:

![image](https://cloud.githubusercontent.com/assets/4684370/15294354/8dcda968-1b94-11e6-98e9-efaaf3eb3b91.png)

**Comment by louis-eyond** [comment]
I've been getting the "Maximum call stack size exceeded" error message as well.  Mine have been happening on some of the Dashboards, different ones with different questions.  Chrome, Metabase 0.17.

I am not sure the exact steps to get into this state, but it appears like it might be after moving around charts and resizing them multiple times.

Here is the screenshot:
![image](https://cloud.githubusercontent.com/assets/816674/16323769/5e75544a-3962-11e6-80c0-a2900e22c563.png)

**Comment by camsaul** [comment]
@HelmiRifai @lewisl9029 are you still seeing this issue?

**Comment by louis-eyond** [comment]
I am not seeing it in the v0.18.1 build.  Thanks

**Comment by camsaul** [comment]
@HelmiRifai?

**Comment by argenisleon** [comment]
I am getting this erro too.

![image](https://cloud.githubusercontent.com/assets/37144/17754890/6301d8ca-649c-11e6-9bc0-cec5e2a3daa9.png)

**Comment by jchin17** [comment]
Just got this error but fully quitting Chrome and restarting it got things working again.

**Comment by tlrobinson** [comment]
Resolved by #4071

@jchin17 @argenisleon @louis-eyond @HelmiRifai, could you please verify this is fixed for you on `master`, if you have a chance.

**Comment by slash-cyberpunk** [comment]
Not a fix...
![image](https://user-images.githubusercontent.com/2331028/30054272-36289cfa-9234-11e7-8bf1-51d96bbccc10.png)

**Comment by vitorfraga** [comment]
I have the same problem.

Your browser and the version: Chrome 59.0.3071.115
Your operating system: Ubuntu 15.04
Your databases: PostgresSQL v9.5.4
Metabase version: v0.25.2
Metabase hosting environment: Ubuntu 16.04
Metabase internal database: PostgreSQL v9.6.3


![image](https://user-images.githubusercontent.com/4948363/30220882-9e3f9db0-9497-11e7-8d11-f9eab54b2b66.png)
![image](https://user-images.githubusercontent.com/4948363/30220901-b4419050-9497-11e7-9815-7c15f1e9b43f.png)

**Comment by RWiggers** [comment]
Same problem here

![image](https://user-images.githubusercontent.com/11998265/32285965-20c436ba-bf13-11e7-8a9a-fe874f4a0f6a.png)

**Comment by nicolinux** [comment]
I have the same problem with Chrome and Metabase 0.26.1.

**Comment by ssilberman** [comment]
Hi, same problem here! Chrome version 62.0.3202.89 (Build oficial) (64 bits) and Metabase 0.26.2

**Comment by mottosso** [comment]
Also seeing this.

Here's what I get in the console.

```js
Refused to load the script 'data:application/javascript;base64,KGZ1bmN0aW9uKCkgewoJLy8gaHR0cHM6Ly9kZXZlbG9wZXJzLmdvb2dsZS5jb20vYW5hbHl0aWNzL2Rldmd1aWRlcy9jb2xsZWN0aW9uL2FuYWx5dGljc2pzLwoJdmFyIG5vb3BmbiA9IGZ1bmN0aW9uKCkgewoJCTsKCX07Cgl2YXIgbm9vcG51bGxmbiA9IGZ1bmN0aW9uKCkgewoJCXJldHVybiBudWxsOwoJfTsKCS8vCgl2YXIgVHJhY2tlciA9IGZ1bmN0aW9uKCkgewoJCTsKCX07Cgl2YXIgcCA9IFRyYWNrZXIucHJvdG90eXBlOwoJcC5nZXQgPSBub29wZm47CglwLnNldCA9IG5vb3BmbjsKCXAuc2VuZCA9IG5vb3BmbjsKCS8vCgl2YXIgdyA9IHdpbmRvdywKCQlnYU5hbWUgPSB3Lkdvb2dsZUFuYWx5dGljc09iamVjdCB8fCAn...0gY2F0Y2ggKGV4KSB7CgkJfQoJfTsKCWdhLmNyZWF0ZSA9IGZ1bmN0aW9uKCkgewoJCXJldHVybiBuZXcgVHJhY2tlcigpOwoJfTsKCWdhLmdldEJ5TmFtZSA9IG5vb3BudWxsZm47CglnYS5nZXRBbGwgPSBmdW5jdGlvbigpIHsKCQlyZXR1cm4gW107Cgl9OwoJZ2EucmVtb3ZlID0gbm9vcGZuOwoJd1tnYU5hbWVdID0gZ2E7CgkvLyBodHRwczovL2dpdGh1Yi5jb20vZ29yaGlsbC91QmxvY2svaXNzdWVzLzMwNzUKCXZhciBkbCA9IHcuZGF0YUxheWVyOwoJaWYgKCBkbCBpbnN0YW5jZW9mIE9iamVjdCAmJiBkbC5oaWRlIGluc3RhbmNlb2YgT2JqZWN0ICYmIHR5cGVvZiBkbC5oaWRlLmVuZCA9PT0gJ2Z1bmN0aW9uJyApIHsKCQlkbC5oaWRlLmVuZCgpOwoJfQp9KSgpOw==' because it violates the following Content Security Policy directive: "script-src 'unsafe-inline' 'unsafe-eval' 'self' https://maps.google.com https://apis.google.com https://www.google-analytics.com https://*.googleapis.com *.gstatic.com ".

app-main.bundle.js:213029 RangeError: Maximum call stack size exceeded
    at a (inflection.js?70ad:1)
    at i (inflection.js?70ad:1)
    at a (inflection.js?70ad:1)
    at h._reconcilerUpdateChildren (inflection.js?70ad:1)
    at h._updateChildren (inflection.js?70ad:1)
    at h.updateChildren (inflection.js?70ad:1)
    at h._updateDOMChildren (inflection.js?70ad:1)
    at h.updateComponent (inflection.js?70ad:1)
    at h.receiveComponent (inflection.js?70ad:1)
    at Object.receiveComponent (isEventSupported.js?4cb5:11)
```

A few refreshes and it goes away. Then it came back again. Sometimes it freezes the tab entirely, occupying 100% of a core. Recreate the tab, and the message is back.

It only happens when viewing one of my dashboards. Latest Chrome as of today, dashboard contains 8 graphs, all of which coming from a MongoDB 3.4 database.

![image](https://user-images.githubusercontent.com/2152766/32700467-d1cbd302-c7bd-11e7-823c-a4d2ae614760.png)

**Comment by gukras** [comment]
Anew news on this issues? is it solve on 0.26?

**Comment by codewinkel** [comment]
I still have this prob with chromium v60 in raspbian

**Comment by allansene** [comment]
Same problem here, using Chrome. But, when I tried on Safari, it works.

**Comment by ipeevski** [comment]
One more report (for a dashboard, after editing and saving it)

```
app-main.bundle.js:88696 RangeError: Maximum call stack size exceeded
    at Object.scrollMonitor (vendor.bundle.js:86087)
    at Object._updateRootComponent (vendor.bundle.js:86098)
    at Object._renderSubtreeIntoContainer (vendor.bundle.js:86186)
    at Object.renderSubtreeIntoContainer [as unstable_renderSubtreeIntoContainer] (vendor.bundle.js:86154)
    at WindowModal._renderPopover (app-main.bundle.js:12521)
    at WindowModal.componentDidUpdate (app-main.bundle.js:12472)
    at CallbackQueue.notifyAll (vendor.bundle.js:83761)
    at ReactReconcileTransaction.close (vendor.bundle.js:113377)
    at ReactReconcileTransaction.closeAll (vendor.bundle.js:62886)
    at ReactReconcileTransaction.perform (vendor.bundle.js:62833)
```

It was fine after logging out and logging back in.

**Comment by dirnbeja** [comment]
I experience the same message, it seems to be (mostly) reproducible: reloading the page gives me the stack size message, navigation to the dashboard in question "from home" (through clicking) gets me the actual result. Safari 13.0.1 on macOS 10.14.6.

**Comment by flamber** [comment]
@dirnbeja There was a PR merged yesterday (#11393), which fixes several "Maximum call stack size exceeded" bugs - mostly triggered by zoom/resolution and pie charts, it seems. Will be fixed in 0.34.0
